plugins/readme/api_key.go

```diff
@@ -0,0 +1,86 @@
+package readme
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://docs.readme.com/main/reference/intro/authentication"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Username,
+				MarkdownDescription: "ReadMe user to log in as.",
+				Secret:              false,
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.APIKey,
+				MarkdownDescription: "API Key used to authenticate to ReadMe.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Prefix: "rdme_",
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
```

Comment:
Just wonder if the `APIKey` has a fixed length? It is worth adding it to the composition if so.